src/components/footer.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+export function Footer({
+  user,
+  loggedIn = false,
+  onLogIn,
+  onLogOut,
+}) {
+  // TODO: Find out how to best do error handling
+  const footer = (
+    <footer className="footer">
+      <div className="container-fluid">
+        <div className="row">
+          <div className="col">
+            <p>
+              Data Hub designed and maintained by the MoTrPAC
+              BioInformatics Center at
+              <a href="https://www.stanford.edu/" target="_new"> Stanford University</a>
+            </p>
+            <p>
+              Funded by the
+              <a href="https://commonfund.nih.gov/" target="_new"> NIH Common Fund</a>
+            </p>
+          </div>
+          <div className="col copyright">
+            <p>&#169; XXXX 2018</p>
```

Comment:
If the copyright info is actually needed, then I would suggest avoiding the use of static _year_.
```suggestion
            <p>&#169; XXXX {getCopyrightYear()}</p>
```
```
function getCopyrightYear() {
  const today = new Date();
  const year = today.getFullYear();
  return year;
}
```

Reply:
Implemented! This makes a lot of sense.